packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksContent.tsx

```diff
@@ -198,35 +197,22 @@ const baseRenderLeaf = (props: RenderLeafProps, modifiers: ModifiersStore) => {
   return <span {...props.attributes}>{wrappedChildren}</span>;
 };
 
-type baseRenderElementProps = {
+type BaseRenderElementProps = {
```

Comment:
I still think this should be an interface, at least for consistency with the rest of the codebase